### ISSUE TITLE
[2.0.4] Admin `customFields` overlap the `remove`, `change` will be hideen

### ISSUE THREAD

**Issue by tianyingchun** [vendure-ecommerce/vendure#2320]
**Describe the bug**
<img width="1023" alt="image" src="https://github.com/vendure-ecommerce/vendure/assets/2833251/c8f71951-dc96-4be8-ae38-23cb7577b932">

**To Reproduce**

please see above picture shown, the small picture it will have `remove` button, if image is big (600*1200) the button will be `hidden`

**Expected behavior**

**Environment (please complete the following information):**
 - @vendure/core version: 2.0.4
 - Nodejs version: 18
 - Database (mysql/postgres etc): mysql

**Additional context**

**Comment by oidt** [comment]
Can you add the customFields config to reproduce this?

**Comment by tianyingchun** [comment]
```ts
productVariantCustomFields

  {

    type: 'relation',
    name: 'visualFrameAsset',
    nullable: true,
    entity: Asset,
    eager: true,
    public: true,
  },
```

**Comment by tianyingchun** [comment]
yes i use `AssetServerPlugin` plugin , but my CDN don't support `preview=tiny`
